components/compose/browser-toolbar/src/main/java/mozilla/components/compose/browser/toolbar/BrowserToolbar.kt

```diff
@@ -49,25 +49,37 @@ fun BrowserToolbar(
         observe = { tab -> tab?.content?.url }
     )
 
+    val items = listOf(
+        BrowserToolbarActionMenuItem("Settings") { onSettingMenuClicked() },
+        BrowserToolbarActionMenuItem("Find in page") {},
+        BrowserToolbarActionMenuItem("Add to homescreen") {}
+    )
+
     val url = selectedTab?.content?.url ?: ""
     val input = when (editText) {
         null -> url
         else -> editText
     }
 
-    if (editMode) {
-        BrowserEditToolbar(
-            url = input,
-            onUrlCommitted = { text -> onTextCommit(text) },
-            onUrlEdit = { text -> onTextEdit(text) }
-        )
-    } else {
-        BrowserDisplayToolbar(
-            url = selectedTab?.content?.url ?: hint,
-            onUrlClicked = {
-                onDisplayToolbarClick()
-            },
-            onMenuClicked = { onDisplayMenuClicked() }
-        )
-    }
+    TopAppBar(
```

Comment:
`TopAppBar` works for simple toolbars. But probably not for a complex one like the one in Fenix? After all the purpose of `BrowserToolbar` is to provide something like `TopAppBar`, but for browsers. 🤔